src/app/routes/article/getInitialData/index.ts

```diff
@@ -74,16 +75,14 @@ export default async ({
       logger.error('Recommendations JSON malformed', error);
     }
 
-    const { topStories, features, mostRead, mostWatched } = secondaryData;
+    const { mostRead, mostWatched } = secondaryData;
+    const secondaryColumn = omit(['mostRead', 'mostWatched'], secondaryData);
 
     const response = {
       status,
       pageData: {
         ...article,
-        secondaryColumn: {
-          topStories,
-          features,
-        },
+        secondaryColumn,
```

Comment:
Not sure if its any more readable, but you could do this to not have to use Ramda:
```
const { mostRead, mostWatched, ...rest } = secondaryData;

pageData: {
  ...article,
  secondaryColumn: {
    ...rest
  }
},
```

`...rest` can be called anything though, so maybe something more applicable to secondaryData. I think the only missing one from the original destructuring was `latestMedia`, so you could probably keep it the way it was and just add that extra object in?

Reply:
The way I did it previously was to make sure that we didn't have duplication of data in both the secondary column & the pageData object - especially with mostRead being so verbose. 

So I guess we have a few options:
1. Leave the implementation as is
1. Spread the secondaryData object like you have suggested (and get rid of rambda's `omit` function)
1. Be explicit about which fields to include in the secondary column - meaning if we add new things in the BFF, then we will also have to make the corresponding change in Simorgh...

Should we have a vote?

Reply:
I would agree. I've made the change now 👍